cmd/cli/kubectl-kyverno/commands/create/templates/aggregated-role.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kyverno-{{.ResourceType}}-permission
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-kyverno: "true"
```

Comment:
This does not seem like the correct label. 

In 1.13 the aggregation labels were updated to:

https://main.kyverno.io/docs/installation/customization/#customizing-permissions

In prior releases, they required 3 labels from the Helm installation:

https://kyverno.io/docs/installation/customization/#customizing-permissions

We can focus on 1.13+ and document that in the help.